test/clj/game_test/cards/programs.clj

```diff
@@ -511,6 +511,24 @@
         (is (zero? (get-counters (refresh c) :virus)) "Consume loses counters")
         (is (zero? (get-counters (refresh h) :virus)) "Hivemind loses counters")))))
 
+(deftest ^:test-refresh/focus corroder
```

Comment:
:eyes: 
Btw. I've got these mappings to deal with that quickly (`<leader>no` for adding and `<leader>ni` for removing the test-refresh thingy). Maybe you'll find it useful as well!
```
map <leader>no mZ?deftest<cr>wi^:test-refresh/focus <esc>'ZmZ
map <leader>ni mZ?deftest<cr>wdf <esc>'ZmZ
```